src/priority/priority.py

```diff
@@ -315,6 +315,9 @@ def reprioritize(self,
         :param exclusive: (optional) Whether this stream should now be an
             exclusive dependency of the new parent.
         """
+        if stream_id == 0:
+            raise MissingStreamError("Stream 0 is non-existent")
```

Comment:
Let's expand this a bit and say "Cannot reprioritize stream 0." I think that's a bit clearer. Do we maybe want to add a new exception for this?